File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.py
import sys
from collections import deque
input=sys.stdin.readline

def fibo(k):
    if k==0:
        return 0
    elif k<=2:
        return 1
    return fibo(k-1) + fibo(k-2)


n= int(input().rstrip())
print(fibo(n))

File: 백준/Bronze/10870. 피보나치 수 5/피보나치 수 5.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : 'C:/psw/python/input.txt'; 
let num = fs.readFileSync(filePath).toString().trim(); 

function fibo(num) {
    if (num == 0) 
        return 0;
    else if (num <= 2) {
        return 1;
    }
    return fibo(num - 1) + fibo(num - 2);
}

console.log(fibo(num))